src/Symfony/Component/Validator/Tests/Constraints/EqualToValidatorTest.php

```diff
@@ -54,6 +54,14 @@ public function provideValidComparisons()
     /**
      * {@inheritdoc}
      */
+    public function provideValidComparisonsToPropertyPath()
+    {
+        yield array(5);
```

Comment:
Why not use `return array(5);`. That would be more similar to what we already have and would avoid questions about why the new ones are different.